### ISSUE TITLE
About Mean.detach()

### ISSUE THREAD

**Issue by Guan2014** [BIT-DA/SDCA#4]

**Comment by Guan2014** [comment]
Thanks for your inspia work!

**Comment by Guan2014** [comment]
Thanks for your impressive work!
I noticed that the estimated mean and covariance are detached in your code. The estimated mean at t-th image includes two components: the estimated mean at (t−1)-th image and the mean of t-th image. And I am wondering how it would be if only the estimated mean and covariance at (t−1)-th image are detached. Could you please explain the difference?

**Comment by BinhuiXie** [comment]
Hi @Guan2014 . Thanks for your comments!
Actually, we consider the estimated mean and covariance as supervisory signals for per-pixel discriminative representation learning, during the training, therefore, they are detached.

**Comment by Guan2014** [comment]
Ok, thank you.
